### DISCUSSION TITLE
[Webserver]: Load web files (html, css, js) from flash

### DISCUSSION THREAD

**Discussion by hasenradball** [micropython/micropython#11513] [General]
Hello Team :-),

from Arduino core of ESP8266 I know that it is possible to store the web files (html, js, css) in the flsh and load it when needed.
(method server.on(....))

Is this also possible in micropython, that you can program your web files and stor it in the flash?

Best regards
frank

**Comment by jimmo** [comment]
Yes. MicroPython provides access to the flash via the filesystem, so you can store your files as...well...files.

If you are using `microdot` as your web server (it's good), then you can just use `send_file` in your handler. See https://microdot.readthedocs.io/en/latest/intro.html#file-responses
